### PR TITLE
use oskarstark/behat-contexts for purging and fixtures

### PR DESCRIPTION
**Pull request by OskarStark**
I am targeting this branch because this is BC.

## Changelog

```markdown
### Changed
- use external lib `oskarstark/behat-contexts` for loading fixtures and purging database
```    
## To do
    
    - [ ] FIX: You have requested a non-existent service "Doctrine\ORM\Tools\SchemaTool".

### CODE REVIEW

**Comment by OskarStark** [comment]
Can you help?

File: composer.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "behat/symfony2-extension": "^2.1",
     "doctrine/doctrine-fixtures-bundle": "^3.0",
```
Comment (kunicmarko20) [inline composer.json:40]: is this needed anymore?

**Comment by kunicmarko20** [comment]
are you sure "Doctrine\ORM\Tools\SchemaTool" is included in composer? since I only required Fixtures, not the orm

**Comment by kunicmarko20** [comment]
Tbh, not really a fan of:

```
  Background:
    Given I am on the dashboard
    And the following YAML fixtures where loaded:
    """
    App\Entity\Category:
        category_1:
            name: 'Dummy Category'
    """
    When I follow "Category"
```

It is exposing too much tech stuff to the reader, but I will merge it if you make it green

**Comment by OskarStark** [comment]
You are right, its a bit too much for these small tests

**Pull request closed without merging**